packages/ui-react/src/components/CollapsibleText/CollapsibleText.tsx

```diff
@@ -11,25 +10,21 @@ import styles from './CollapsibleText.module.scss';
 type Props = {
   text: string;
   className?: string;
-  maxHeight?: 'none' | number;
 };
 
-const CollapsibleText: React.FC<Props> = ({ text, className, maxHeight = 'none' }: Props) => {
+const CollapsibleText: React.FC<Props> = ({ text, className }: Props) => {
   const divRef = useRef<HTMLDivElement>() as React.MutableRefObject<HTMLDivElement>;
-  const breakpoint = useBreakpoint();
```

Comment:
We might need the `useBreakpoint` here to update the `doesFlowOver` state when the browser resizes. It probably works now because the `TruncatedText` also uses the `useBreakpoint` hook, but not when this component is used directly.

Reply:
Sounds good, I've reintroduced it!